profiles/common/modules/features/ec_embedded_video/includes/ec_embedded_video.formatters.inc

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * @file
+ * File formatters for Platform needs.
+ */
+
+/**
+ * Implements hook_file_formatter_info_alter().
+ *
+ * It alters file_formatter_info to add the "Wrapper free formatters for the
+ * different Video-sharing services (YouTube, Vimeo, Dailymotion and
+ * AV portal).
+ */
+function ec_embedded_video_file_formatter_info_alter(&$info) {
```

Comment:
@drupol, I was searching another way than the alter hook but when I see the code, I think that that, in this case, the alter hook is cached, see field.info.inc line 210 where the hook alter is called; it is in the process part called when the cache record is not found only.